### DISCUSSION TITLE
can't connetc via Bt dongle the blueretro gamec

### DISCUSSION THREAD

**Discussion by rafaeoz** [darthcloud/BlueRetro#563] [Q&A]

**Comment by darthcloud** [comment]
Provide a better description of what you are trying to do.
.Connect a controller? Connect to web config?

What kind of BlueRetro you got?

**Reply by rafaeoz** [reply]
I have a bt dongle to use Bluetooth on pc. But with bluretro its not working.
Now i could connect with my Android phone in the browser.So  Its ok. Thanks!
